examples/controls_example/public/app/react_control_example.tsx

```diff
@@ -263,6 +270,28 @@ export const ReactControlExample = ({
             }}
           />
         </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButton
+            size="s"
+            color="danger"
+            isSelected={simulateErrorState}
+            fill={simulateErrorState}
+            iconType={simulateErrorState ? 'errorFilled' : 'error'}
+            onClick={() => {
+              setSimulateErrorState((isOn) => !isOn);
+              Object.keys(controlGroupApi?.children$.getValue() ?? {}).forEach((controlId) => {
+                const control = controlGroupApi?.children$.getValue()[controlId];
+                if (control && (control as DefaultControlApi).setBlockingError) {
+                  (control as DefaultControlApi).setBlockingError(
+                    !simulateErrorState ? new Error('This is a simulated error') : undefined
+                  );
+                }
+              });
```

Comment:
This shouldn't be possible, but we are exposing the blocking error setter via the `DefaultControlApi` - as part of the final refactor steps, we will remove this and no longer expose the setter. See for the relevant TODO comment https://github.com/elastic/kibana/pull/186656/files#diff-017009740ee785b16cae9b05a4c65dcefc2b4706335de7622d9b15e78273bb79R48